kotlin-mbedtls/src/main/kotlin/org/opencoap/ssl/transport/DtlsServer.kt

```diff
@@ -138,18 +143,27 @@ class DtlsServer(
         updateSessionAuthenticationContext(adr, ctx.authenticationContext)
     }
 
-    fun loadSession(sessBuf: SessionWithContext?, adr: InetSocketAddress, cid: ByteArray): Boolean {
+    fun loadSession(sessBuf: SessionWithContext?, adr: InetSocketAddress, cid: ByteArray, buf: ByteBuffer? = null): Boolean {
```

Comment:
We should require `buf` to be present, also rename it, for example:
```suggestion
    fun loadSession(sessBuf: SessionWithContext?, adr: InetSocketAddress, cid: ByteArray, dtlsPacket: ByteBuffer): Boolean {
```

Reply:
The DtlsChannelHandler has a loadSession function which uses this func, what to pass there?
https://github.com/open-coap/kotlin-mbedtls/blob/65adba3972ee2956e99d726a9e7bf2098c5125cf/kotlin-mbedtls-netty/src/main/kotlin/org/opencoap/ssl/netty/DtlsChannelHandler.kt#L78

Reply:
You would need to pass a msg: `msg.content().nioBuffer()`